src/renderer/js/redux/reducers/app.js

```diff
@@ -1,22 +1,45 @@
-import * as types from "constants/action_types";
-import * as modalTypes from "constants/modal_types";
+// @flow
 
+import * as types from "../../constants/action_types";
+import * as modalTypes from "../../constants/modal_types";
+
+// $FlowIssue: Flow cannot find election
 const { remote } = require("electron");
+
 const application = remote.app;
 const win = remote.BrowserWindow.getFocusedWindow();
 
 const reducers = {};
-const defaultState = {
+type appState = {
+  isLoaded: boolean,
+  modal: mixed,
+  modalProps: mixed,
```

Comment:
Isn't this an unsealed `Object` or `null`? (not clear on Flow syntax for this: `?{}`?)

Reply:
I believe mixed is the correct way for to indicate Object

Reply:
@liamcardenas I do not think that is correct https://flow.org/en/docs/types/mixed/